ansible/deploy_proxy.yml

```diff
@@ -25,6 +25,7 @@
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.wiki == True, action: site, site_name: wiki_http }
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech == True, action: site, site_name: motech}
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech == True, action: site, site_name: motech_http }
+    - { role: nginx, when: proxy_type == 'nginx' and active_sites.cchq_ssl_j2me == True, action: site, site_name: cchq_ssl_j2me }
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.commtrack_ssl == True, action: site, site_name: commtrack_ssl }
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.riakcs == True, action: site, site_name: riakcs }
```

Comment:
yeah for now OK to leave. it sets up the config for proxying to riakcs. not used on prod/india, just staging